pkg/resource/detector/README.md

```diff
@@ -0,0 +1,53 @@
+### Example usage:
+
+Setting up the detector:
+```
```

Comment:
```suggestion
```go
```

^this will add correct syntax highlighting to the rendered blocks.

Also there seems to be a mix of spaces and tabs in these blocks for indentation, which is making things like comments look over/under-indented.